src/config.erl

```diff
@@ -208,7 +208,7 @@ handle_call(all, _From, Config) ->
     {reply, Resp, Config};
 handle_call({set, Sec, Key, Val, Persist, Reason}, _From, Config) ->
     true = ets:insert(?MODULE, {{Sec, Key}, Val}),
-    couch_log:notice("~p: [~s] ~s set to ~s for reason ~p",
+    error_logger:info_msg("~p: [~s] ~s set to ~s for reason ~p",
```

Comment:
@kxepal I can use either `lager` or `couch_log` if `couch_log` is shipped as an independent app that I can include in the deps using rebar, but as the app was before I started, I had no access to any `couch_log` module, so it failed.